content/courses/program-optimization/rust-macros.md

```diff
@@ -147,7 +147,7 @@ use syn::parse_macro_input;
 #[proc_macro]
 pub fn my_macro(input: TokenStream) -> TokenStream {
     let ast = parse_macro_input!(input as syn::LitStr);
-    eprintln! {"{:#?}", ast};
+    eprintln!("{:#?}", ast);
```

Comment:
Since syn is already being imported in the use section, LitStr can be imported directly in order to create a cleaner use section